allennlp/common/file_utils.py

```diff
@@ -641,8 +641,8 @@ def __init__(
             str(filename),
             subdir=False,
             map_size=map_size,
-            max_readers=cpu_count * 2,
-            max_spare_txns=cpu_count * 2,
+            max_readers=cpu_count * 4,
```

Comment:
Should we have an option to specify the kwargs in `TensorCache`'s init? Or do we want to shield the users from the gory details?

Reply:
I wanted to shield users from this. It seems like an implementation detail. Another implementation might not have that setting at all.

Reply:
Makes sense.